main.js

```diff
@@ -27,6 +41,15 @@
     }
   }
 
+  function enableUpdateButton() {
```

Comment:
I appreciate that we are getting to repeat some previously expressed logic into new functions that do similar things for the repetition.